ros_gz_shims/ros_ign_gazebo_demos/launch/image_bridge.launch.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Instead of repeating the contents of the launch files, how about just calling the `gz` one like this?

```py
import os

from ament_index_python.packages import get_package_share_directory

from launch import LaunchDescription
from launch.actions import IncludeLaunchDescription
from launch.launch_description_sources import PythonLaunchDescriptionSource


def generate_launch_description():

    pkg_ros_gz_sim_demos = get_package_share_directory('ros_gz_sim_demos')

    return LaunchDescription([
        IncludeLaunchDescription(
            PythonLaunchDescriptionSource(
                os.path.join(pkg_ros_gz_sim_demos, 'launch', 'image_bridge.launch.py')),
        )
    ])
```

This way when we change the `gz` launch file, the shim is updated automatically.

I checked that the arguments are being forwarded properly, for example:

```
ros2 launch ros_ign_gazebo_demos image_bridge.launch.py rqt:=false
```

It would be cool to print a warning, but I haven't looked into how to do it.

Reply:
https://github.com/gazebosim/ros_gz/pull/281/commits/c79ba8bc3ccc36eba15084a1c378a3f08e4fffa4 :>